awx/main/tasks.py

```diff
@@ -2857,8 +2857,13 @@ def _reconstruct_relationships(copy_mapping):
 @task(queue=get_local_queuename)
```

Comment:
Note that this task *always* runs on the local host where it is *published*, so we can lean on the local memcached instance to stash and look up this (large) data structure.